src/psyclone/psyad/transformations/preprocess.py

```diff
@@ -104,7 +108,8 @@ def associativity(assignment, active_variable_names):
     x*b and a*/x +- b*/x respectively.
 
     This function can be removed when support for Range nodes is added
-    to the SymbolicMaths expand function, see issue #1655.
+    to the SymbolicMaths expand function, see issue #1655, and if the
+    expand function can be modified so that it expands literals.
 
     :param assignment: the Assignment Node that we are looking at.
     :type assignment: :py:class:`psyclone.psyir.nodes.Assignment`
```

Comment:
Since we're here, pylint complains about an unnecessary `elif` at L123. Please could you take a look?

Reply:
Fixed.